appintro/src/main/java/com/github/appintro/AppIntro.kt

```diff
@@ -163,6 +162,18 @@ abstract class AppIntro : AppIntroBase() {
         nextButton.setImageDrawable(imageNextButton)
     }
 
+    /**
+     * Override done and skip buttons text size
+     *
+     * @param sizeText your text size from resource (in sp)
+     */
+    fun setTextSizeButtons(@DimenRes sizeText: Int) {
```

Comment:
This should be split in two methods to be consistent with the rest of the API.

Moreover please change this to accept a TextApperance:

`fun setDoneTextAppearance(@StyleRes textAppearance) {`
`fun setSkipTextAppearance(@StyleRes textAppearance) {`

Reply:
I thought about changing the appearance altogether at first so I will do that. Separating them makes sense from that point of view, although I don't believe people are going to change their styles separately

Reply:
> although I don't believe people are going to change their styles separately

Agree, but at the same time we offer both `setDoneTextTypeface` and `setSkipTextTypeface` so we should keep the API consistent on this front.